src/pycmor/core/config.py

```diff
@@ -359,121 +359,40 @@ class Config:
-            parser=float,
-        )
-        xarray_open_mfdataset_engine = Option(
-            default="netcdf4",
-            doc="Which engine to use for xarray.open_mfdataset().",
-            parser=ChoiceOf(
-                str,
-                choices=[
-                    "netcdf4",
-                    "h5netcdf",
-                    "zarr",
-                ],
-            ),
-        )
-        xarray_open_mfdataset_parallel = Option(
-            default="yes",
-            doc=(
-                "Whether to use parallel processing when opening multiple files "
-                "with xarray.open_mfdataset(). Default is True."
-            ),
-            parser=_parse_bool,
-        )
-        xarray_skip_unit_attr_from_drv = Option(
-            default="yes",
-            doc="Whether to skip setting the unit attribute from the DataRequestVariable, this can be handled via Pint",
-            parser=_parse_bool,
-        )
-        xarray_time_dtype = Option(
-            default="float64",
-            doc="The dtype to use for time axis in xarray.",
-            parser=ChoiceOf(
-                str,
-                choices=[
-                    "float64",
-                    "datetime64[ns]",
-                ],
-            ),
-        )
-        xarray_time_enable_set_axis = Option(
-            default="yes",
-            doc="Whether to enable setting the axis for the time axis in xarray.",
-            parser=_parse_bool,
-        )
-        xarray_time_remove_fill_value_attr = Option(
-            default="yes",
-            doc="Whether to remove the fill_value attribute from the time axis in xarray.",
-            parser=_parse_bool,
-        )
-        xarray_time_set_long_name = Option(
+        # NOTE: xarray_* options are dynamically generated by @_generate_xarray_options decorator
+        # See XARRAY_OPTIONS structure above for definitions
+
+        # Coordinate and dimension mapping options
+        xarray_set_coordinate_attributes = Option(
             default="yes",
-            doc="Whether to set the long name for the time axis in xarray.",
+            doc="Whether to set CF-compliant attributes (standard_name, axis, units) on coordinate variables.",
             parser=_parse_bool,
         )
-        xarray_time_set_standard_name = Option(
+        xarray_set_coordinates_attribute = Option(
             default="yes",
-            doc="Whether to set the standard name for the time axis in xarray.",
+            doc="Whether to set the 'coordinates' attribute on data variables listing their coordinates.",
             parser=_parse_bool,
         )
-        xarray_time_taxis_str = Option(
-            default="T",
-            doc="Which axis to set for the time axis in xarray.",
+        xarray_validate_coordinate_attributes = Option(
+            default="warn",
+            doc="Validation mode for existing coordinate metadata. Options: ignore, warn, error, fix",
             parser=str,
```

Comment:
This should be `choice_of` or something like that. [`everett`](https://everett.readthedocs.io/en/latest/) (which is what we use for configuration) has a [mechanism for that](https://everett.readthedocs.io/en/latest/parsers.html#choiceof-parser-list-of-choices):


```suggestion
            default="warn",
            doc="Validation mode for existing coordinate metadata. Options: ignore, warn, error, fix",
            parser=ChoiceOf(str, choices=["warn", "error", "fix"]),
```